osu.Game/Scoring/ScoreInfo.cs

```diff
@@ -31,6 +31,9 @@ public class ScoreInfo : IHasFiles<ScoreFileInfo>, IHasPrimaryKey, ISoftDelete,
         [Column(TypeName = "DECIMAL(1,4)")]
         public double Accuracy { get; set; }
 
+        [JsonIgnore]
+        public string DisplayAccuracy => Accuracy == 1 ? "100%" : $"{Accuracy:P2}";
```

Comment:
`:P2` will actually insert a space before the percent sign in the invariant culture (so i.e. `62.94 %`), which is super strange IMO, and at least inconsistent with the special check for `1`. Therefore I'd say it'd be better to explicitly get rid of that behaviour:
```suggestion
        public string DisplayAccuracy => Accuracy == 1 ? "100%" : $"{Accuracy:0.00%}";
```